clients/admin-ui/src/types/api/models/ClassificationStatus.ts

```diff
@@ -3,7 +3,14 @@
 /* eslint-disable */
 
 /**
- * The model housing statuses for a classify instance
+ * The model housing statuses for a classify instance.
+ *
+ * CREATED: The initial state of an attempt to classify
+ * PROCESSING: Attempting to classify dataset(s)
+ * COMPLETED: The classify process completed successfully
```

Comment:
Nit: guessing this came from autogeneration but "COMPLETED" ≠ "COMPLETE" LOL

Reply:
fixed here, the source comment (on the backend in fidesctl+) will be done separately